src/routes/resources/kendo-japanese-pronunciations-and-definitions/+page.svelte

```diff
@@ -0,0 +1,759 @@
+	</tr>
+	<tr>
+		<th scope="row">e え</th>
+		<td>ke け</td><td>se せ</td><td>te て</td><td>ne ね</td><td>he/e へ</td>
+		<td>me め</td><td> </td><td>re れ</td><td> </td><td> </td>
+	</tr>
+	<tr>
+		<th scope="row">o お</th>
+		<td>ko こ</td><td>so そ</td><td>to と</td><td>no の</td><td>ho ほ</td>
+		<td>mo も</td><td>yo よ</td><td>ro ろ</td><td>wo を</td><td> </td>
+	</tr>
+</table>
+<p>Here are some example English words that correspond to one letter from each column and row:</p>
+<table>
+	<tr>
+		<td><b>a</b>pply</td><td><b>ke</b>el</td><td><b>so</b>on</td><td><b>te</b>nnis</td>
+		<td><b>kno</b>ll</td><td><b>ha</b>lt</td><td><b>me</b></td><td><b>you</b></td>
+		<td><b>re</b>sult</td><td><b>wa</b>lk</td><td><b>n</b>o</td>
+	</tr>
+</table>
+<p>
+	Several columns are characters that are further modified by the addition of quotes or a small
+	circle – both are placed in the upper right corner of the character they modify. This changes the
+	consonants at the beginning of the character respectively:
+</p>
+<table dir="ltr">
+	<tr>
+		<td> </td>
+		<th scope="col">“G”</th><th scope="col">“Z”</th><th scope="col">“D”</th>
+		<th scope="col">“B”</th><th scope="col">“P”</th>
+	</tr>
+	<tr>
+		<th scope="row">“a”</th>
+		<td>ga が</td><td>za ざ</td><td>da だ</td><td>ba ば</td><td>pa ぱ</td>
+	</tr>
+	<tr>
+		<th scope="row">“i”</th>
+		<td>gi ぎ</td><td>ji じ</td><td>ji ぢ</td><td>bi び</td><td>pi ぴ</td>
+	</tr>
+	<tr>
+		<th scope="row">“u”</th>
+		<td>gu ぐ</td><td>zu ず</td><td>zu づ</td><td>bu ぶ</td><td>pu ぷ</td>
+	</tr>
+	<tr>
+		<th scope="row">“e”</th>
+		<td>ge げ</td><td>ze ぜ</td><td>de で</td><td>be べ</td><td>pe ペ</td>
+	</tr>
+	<tr>
+		<th scope="row">“o”</th>
+		<td>go ご</td><td>zo ぞ</td><td>do ど</td><td>bo ぼ</td><td>po ぽ</td>
+	</tr>
+</table>
+<p>
+	Certain characters may be written small in combination with others to produce additional
+	syllables. Similar to abbreviations, these small characters denote that the sound is to be
+	underaccented than if it were written largely. Some examples are shown below:
+</p>
+<table dir="ltr">
+	<tr>
+		<td> </td>
+		<th scope="col">き “ki”</th><th scope="col">に “ni”</th><th scope="col">ひ “hi”</th>
+		<th scope="col">ま “mi”</th><th scope="col">り “ri”</th><th scope="col">し “shi”</th>
+		<th scope="col">ち “chi”</th>
+	</tr>
+	<tr>
+		<th scope="row">“ya” ゃ</th>
+		<td>kya きゃ</td><td>nya にゃ</td><td>hya ひゃ</td>
+		<td>mya まゃ</td><td>rya りゃ</td><td>sha しゃ</td>
+		<td>cha ちゃ</td>
+	</tr>
+	<tr>
+		<th scope="row">“yu” ゅ</th>
+		<td>kyu きゅ</td><td>nyu にゅ</td><td>hyu ひゅ</td>
+		<td>myu まゅ</td><td>ryu りゅ</td><td>shu しゅ</td>
+		<td>chu ちゅ</td>
+	</tr>
+	<tr>
+		<th scope="row">“yo” ょ</th>
+		<td>kyo きょ</td><td>nyo にょ</td><td>hyo ひょ</td>
+		<td>myo まょ</td><td>ryo りょ</td><td>sho しょ</td>
+		<td>cho ちょ</td>
+	</tr>
+</table>
+<p>Please feel free to explore the web or seek additional instruction/information.</p>
+
+<style>
+	dt {
+		font-weight: 500;
+	}
+	dd {
+		color: var(--pico-secondary);
+	}
+	table {
+		display: block;
```

Comment:
Are you sure you want to change the table into a block?

Reply:
[I got that from MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table#:~:text=in%20this%20example.-,CSS,-When%20looking%20at). Without it, the tables don't have scrollbars. Is there another way to do that?

Reply:
Here's a quote from the article you linked:
> When looking at these styles you'll notice that table's display property has been set to block. While this allows scrolling, the table loses some of its integrity

I think it's probably better to just leave the tables as `display: table` and then wrap the table in a div kinda like the way pico does it.

Reply:
I did notice that. But I also don't super love where the scrollbar gets placed when you have a div around the table (which looks the same with pico's classes or not, so if we're going to do it that way, we might as well use pico for it). It gets really close to the neighboring elements on the bottom and there's a gap between the scrollbar and the actual table

Reply:
Oh yeh that's a good point, I didn't notice that. We should look into fixing fixing the margins.